vignettes/algorithm.Rmd

```diff
@@ -77,7 +77,7 @@ S_i = \begin{pmatrix}
 Note that this follows from the well known property of the multivariate normal
 distribution that linear combinations of the random vector again have a
 multivariate normal distribution with the correspondingly modified mean
-vector and covariance matrix.
+vector and covariance matrix. When weights are specified, we replace the covariance matrix $\Sigma_i$ with $Z_i^{-1/2} \Sigma_i Z_i^{-1/2}$, where $Z_i$ is the diagonal weight matrix.
```

Comment:
maybe it would be easier to read and explain when we always use this more general form? i.e. already in line 60 define $\Sigma_i$ accordingly and introduce what $Z_i$ is. If no weights then it is just the identity matrix.